src/ocrd/mets_server.py

```diff
@@ -428,8 +430,7 @@ def create_process(mets_server_url: str, ws_dir_path: str, log_file: str) -> int
 
     @staticmethod
     def kill_process(mets_server_pid: int):
-        subprocess_run(args=["kill", "-s", "SIGINT", f"{mets_server_pid}"], shell=False, universal_newlines=True)
-        return
+        return os.kill(mets_server_pid, signal.SIGTERM)
```

Comment:
I would still prefer to handle that softly first with `signal.SIGINT` sleep for some time then send `signal.SIGTERM` to give enough time for the Mets Server process to clean stuff after itself (if required).

Reply:
Sure, I'm open to that. Ideally we should test this and see whether SIGINT is enough (I know it was in my testing but since I need to work around this issue today, I want to be certain that process `mets_server_pid` is gone after `kill.process` (and the workaround zombie killer endpoint). `SIGINT` and `SIGTERM` should behave mostly the same anyway, `SIGKILL` would be the harsher solution.

Reply:
Yes, just the idea is to stay flexible and let the process catch the signal and do some internal cleaning. Sorry, I meant the harsher`SIGKILL` instead of `SIGTERM`. Just in case the process still gets stuck during the clean up after catching the SIGINT.